src/plugins/Network.js

```diff
@@ -151,16 +169,28 @@ module.exports = class Network extends OverlayPlugin {
                 this.selectProfile.add(option);
             });
         this.profileInputs.appendChild(this.selectProfile);
+
+        this.mobileThrottlingConfigured = true;
     }
 
     enable5G() {
+        if (this.network5GConfigured) {
```

Comment:
Why don't you just check for existing option 'profile.name', and avoid the addition if it is already present?

Reply:
5G is always preset in mobile profile list we add it to the selector only if the enable5G method is triggered.